src/components/filter/EventsFilterPanel.tsx

```diff
@@ -124,11 +124,8 @@ function EventsFilterPanel() {
 
 			let toggler: FilterRowTogglerConfig | null = null;
 
-			const autocompleteList = getArrayOfUniques(
-				eventsHistory
-					.filter(item => item.filters[filterName])
-					.map(item => item.filters[filterName]?.values)
-					.flat(),
+			const autocompleteList: string[] = getArrayOfUniques(
```

Comment:
this does not fix the problem. autocompleteList is (string | undefined)[], component expects string[] | null